llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -5344,11 +5344,10 @@ InstructionCost AArch64TTIImpl::getPartialReductionCost(
   } else
     return Invalid;
 
-  // AArch64 supports lowering mixed extensions to a usdot but only if the
-  // i8mm or sve/streaming features are available.
+  // AArch64 supports lowering mixed fixed-width extensions to a usdot but only
+  // if the i8mm feature is available.
   if (OpAExtend == TTI::PR_None || OpBExtend == TTI::PR_None ||
-      (OpAExtend != OpBExtend && !ST->hasMatMulInt8() &&
-       !ST->isSVEorStreamingSVEAvailable()))
+      (OpAExtend != OpBExtend && VF.isFixed() && !ST->hasMatMulInt8()))
```

Comment:
I think `VF.isFixed()` can be removed, because SVE also has usdot instructions (if i8mm is available)

Reply:
Done.